MealFridge/Views/Search/RecipeCards.cshtml

```diff
@@ -9,45 +9,53 @@
             Shelved = false,
             Favorited = false,
         };
-        @if (temp.Shelved ?? false)
+        @if (temp.Shelved ?? false || r.Banned)
         {
             continue;
         }
-  <div class="card shadow-lg">
-            <img class="card-img-top" src="@r.Image" alt="Recipe Image">
-                @if (temp.Favorited ?? false)
-                 {
-                    <div class="card-img-overlay">
-                        <h3 class="text-md-right p-1 text-danger bg-dark d-inline shadow-lg">
-                            <i class="fa fa-heart"> </i> 
-                        </h3>
-                    </div>
-                 }
-                 <div class="card-body">
-                     <h4 class="card-title"> @r.Title </h4>
-                 </div>
-                 <div class="card-footer">
-                     <div class="btn-group btn-group-lg w-100">
-                         <button type="button"
-                                 id="favButton"
-                                 class="btn btn-outline-success"
-                                 style="display:inline-block"
-                                 onclick="addFavorite('@r.Id')" title="Favorite recipe">
-                             <i class="fa fa-heart"> @r.Savedrecipes.Where(f => f.RecipeId == r.Id && f.Favorited != false).Count()</i>
-                         </button>
+<div class="card shadow-lg">
+    <img class="card-img-top" src="@r.Image" alt="Recipe Image">
+    @* THERE IS A BUG DO NOT TOUCH UNLESS YOU ARE SMARTER THAN CHRISTIAN @if (temp.Favorited)
```

Comment:
Gonna need more explanation on this one

Reply:
Made a note that there's been a bug from a previous sprint in this logic that every saved recipe will occasionally flag all search recipes with a heart. Commented it out for this sprint in order to show that my restrictions being taken into account works.